uwdata/lyra
# WIP: Scale Component

**Description of the problem this pull request fixes**
This pull build more functionality to scale.

Changes proposed in this pull request:
- I add the functionality of of types of scale.

**This PR includes:**

**Steps to functionally test this:**
- create scales
- go to https://github.com/vega/vega/wiki/Scales and check the properties for Quantitative Scale, time scale, ordinal scale.
- the user should be able to change name
- the user should be able to change type
- the user should be able to play around the properties with all types of scale.
